src/main/java/sg/ncl/MainController.java

```diff
@@ -3494,6 +3486,514 @@ public String adminEnergy(Model model,
+            return NO_PERMISSION_PAGE;
+        }
+
+        HttpEntity<String> request = createHttpEntityHeaderOnly();
+        restTemplate.setErrorHandler(new MyResponseErrorHandler());
+        ResponseEntity response = restTemplate.exchange(properties.getMonthlyUsage(id) + "/" + month, HttpMethod.DELETE, request, String.class);
+        String responseBody = response.getBody().toString();
+
+        try {
+            if (RestUtil.isError(response.getStatusCode())) {
+                MyErrorResource error = objectMapper.readValue(responseBody, MyErrorResource.class);
+                ExceptionState exceptionState = ExceptionState.parseExceptionState(error.getError());
+                switch (exceptionState) {
+                    case PROJECT_USAGE_NOT_FOUND_EXCEPTION:
+                        log.warn("Project usage not found for deleting");
+                        attr.addFlashAttribute(MESSAGE, "Error(s):<ul><li>project usage not found for deleting</li></ul>");
+                        break;
+                    case FORBIDDEN_EXCEPTION:
+                        log.warn("Removing of project usage forbidden.");
+                        attr.addFlashAttribute(MESSAGE, "Error(s):<ul><li>deleting project forbidden</li></ul>");
+                        break;
+                    default:
+                        log.warn("Unknown error for validating project usage.");
+                        attr.addFlashAttribute(MESSAGE, "Error(s):<ul><li>unknown error for deleting project usage</li></ul>");
+                }
+            } else {
+                log.info("Project usage deleted: {}", responseBody);
+            }
+        } catch (IOException e) {
+            log.error("adminMonthlyRemove: {}", e.toString());
+            throw new WebServiceRuntimeException(e.getMessage());
+        }
+
+        return "redirect:/admin/monthly/" + id + "/usage";
+    }
+
+    @GetMapping("/admin/statistics")
+    public String adminUsageStatistics(HttpSession session, Model model) {
+        if (!validateIfAdmin(session)) {
+            return NO_PERMISSION_PAGE;
+        }
+
+        if (!model.containsAttribute("query")) {
+            model.addAttribute("query", new ProjectUsageQuery());
+            model.addAttribute("newProjects", new ArrayList<ProjectDetails>());
+            model.addAttribute("activeProjects", new ArrayList<ProjectDetails>());
+            model.addAttribute("inactiveProjects", new ArrayList<ProjectDetails>());
+            model.addAttribute("utilization", new HashMap<String, MonthlyUtilization>());
+            model.addAttribute("statsCategory", new HashMap<String, Integer>());
+            model.addAttribute("statsAcademic", new HashMap<String, Integer>());
+        }
+
+        return "admin_usage_statistics";
+    }
+
+    @PostMapping("/admin/statistics")
+    public String adminUsageStatisticsQuery(@Valid @ModelAttribute("query") ProjectUsageQuery query, BindingResult result,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 37 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)